### PR TITLE
fix Edit Document

### PR DESCRIPTION
**Pull request by sy-records**
**Summary**

Fix Chinese document edit button error.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [ ] DO NOT include files inside `lib` directory.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
         '.*?/awesome': 'https://raw.githubusercontent.com/docsifyjs/awesome-docsify/master/README.md',
         '.*?/changelog': 'https://raw.githubusercontent.com/docsifyjs/docsify/master/CHANGELOG.md',
         '/.*/_navbar.md': '/_navbar.md',
-        '/zh-cn/(.*)': 'https://cdn.jsdelivr.net/gh/docsifyjs/docs-zh/$1',
+        '/zh-cn/(.*)': 'https://cdn.jsdelivr.net/gh/docsifyjs/docs-zh@master/$1',
```
Comment (anikethsaha) [inline docs/index.html:35]: why `@` ?

Comment (sy-records) [reply docs/index.html:35]: @master is the default, plus it is for easy replacement, is there any better implementation? I am not professional.

Comment (anikethsaha) [reply docs/index.html:35]: ok lets keep it like this for now. I will take a look later if possible 👍

Comment (sy-records) [reply docs/index.html:35]: https://github.com/docsifyjs/docsify/blob/2cf6fd8698af916f1d29b76578f8b63d52028c7a/docs/index.html#L83

**Pull request merged**